### DISCUSSION TITLE
Add option to rotate video on Instagram

### DISCUSSION THREAD

**Discussion by AslanChile** [datarhei/restreamer#347] [Ideas]
Most stream destinations use wide format (16:9), except Instagram which now requires stretched (9:16) and it would be great to be able to send the same video rotated
I understand that FFmpeg allows to rotate without re-encoding
I'm sure many users would use it

**Comment by ioppermann** [comment]
This is true. Depending on the container and/or codec, it is possible to add information about the rotation, resp. orientation, in the metadata. This can be done without re-encoding, e.g. for H.264 it could be done with the `h264_metadata` bitstream filter (`-bsf:v h264_metadata=display_orientation=insert:rotate=90`.

However, it depends on the player/decoder to respect this information or not. If Instagram is respecting this value then it might work.

**Comment by svenerbeck** [comment]
@AslanChile  Added your suggestion on the [feature request list](https://github.com/datarhei/restreamer/discussions/345).

**Comment by AslanChile** [comment]
Thank you

**Comment by svenerbeck** [comment]
Hello @AslanChile 
[https://www.youtube.com/watch?v=wYQeTk6GmB4](https://www.youtube.com/watch?v=wYQeTk6GmB4)
Cheerio
Sven

[Closed]

**Comment by alexisod** [comment]
Is there something similar for cropping the source feed?
